<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFeaturesDlg</class>
 <widget class="QDialog" name="CreateFeaturesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make features</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="src_col_name_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="delta_">
     <property name="text">
      <string>delta</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="delta_2">
       <property name="text">
        <string>delta period 2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="delta_4">
       <property name="text">
        <string>delta period 4</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="delta_3">
       <property name="text">
        <string>delta period 3</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="delta_5">
       <property name="text">
        <string>delta period 5</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateFeaturesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateFeaturesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
